scipy/optimize/__init__.py

```diff
@@ -195,6 +196,8 @@
 +-------------+----------+----------+-----------+-------------+-------------+----------------+
 | `R` or `C`  | No       | Yes      | No        | newton      | No          | 2.00 (1.41)    |
 +-------------+----------+----------+-----------+-------------+-------------+----------------+
+| `R` or `C`  | No       | Yes      | No        | exnewton    | No	    | >2.00 (1.41)   |
++-------------+----------+----------+-----------+-------------+-------------+----------------+
```

Comment:
No need for the line of "-" as the column values for Bracket/fprime/fprime2/Solvers are the same as for `newton`. Similar to the collection of bisect/brentq/.../toms748